docs/topics/feed-exports.rst

```diff
@@ -268,6 +268,38 @@ in multiple files, with the specified maximum item count per file. That way, as
 soon as a file reaches the maximum item count, that file is delivered to the
 feed URI, allowing item delivery to start way before the end of the crawl.
 
+.. _item-filter:
+
+Item filtering
+==============
+
+You can filter items that you want to allow for a particular feed storage by using
+the ``item_classes`` option in :ref:`feeds options <feed-options>`. Add the items you
+wish to accept and only those items will be entered to the feed. This feature is actually
+implemented by Scrapy's :class:`~scrapy.extensions.feedexport.ItemFilter` class which is 
+the default value of ``item_filter`` option in the :ref:`feeds options <feed-options>`.
+
+You can create your own custom filtering class by implementing :class:`~scrapy.extensions.feedexport.ItemFilter`'s
+method ``accepts`` and taking ``feed_options`` as an argument.
+
+For instance::
+
+    class MyCustomFilter:
+
+        def __init__(self, feed_options):
+            self.feed_options = feed_options
+
+        def accepts(self, item):
+            if "field1" in item and item["field1"] == "expected_data":
+                return True
+            return False
```

Comment:
What about this, based on @adityaa30’s feedback during our last call? (the example is untouched, just added a paragraph below)

```suggestion
For instance::

    class MyCustomFilter:

        def __init__(self, feed_options):
            self.feed_options = feed_options

        def accepts(self, item):
            if "field1" in item and item["field1"] == "expected_data":
                return True
            return False
            
You can assign your custom filtering class to the ``item_filter`` :ref:`option 
of  a feed <feed-options>`. See :setting:`FEEDS` for examples.
```